.travis.yml

```diff
@@ -1,20 +1,24 @@
 language: python
-python:
-  - "3.4"
-  - "3.5"
-  - "3.6"
+cache: pip
 matrix:
   include:
+    - python: 3.4
+      name: "Python 3.4 pytest (default linux)"
+    - python: 3.5
+      name: "Python 3.5 pytest & mypy type-checking (default linux)"
+      env: RUN_MYPY=yes
```

Comment:
I found that the Travis trusty image has 3.6 by default, but has to download other versions for 3.4 and 3.5. I think the overall test time would have been reduced if Mypy test is piled on 3.6.

Reply:
https://travis-ci.org/zulip/zulip-terminal/jobs/468316993#L415 it is not logged how long it takes to download python-3.4.tar.bz2. It is possibly ~5s.

Reply:
Yup, I seems like a good idea to run mypy on 3.6 instead of 3.5.

Reply:
The new version has this update.